app/views/ProjectEdit/Framework/index.tsx

```diff
@@ -156,37 +172,36 @@ function ProjectFramework(props: Props) {
     } = useForm(schema, defaultFormValue);
 
     const delayedValue = useDebouncedValue(value);
-
     useEffect(() => {
         setOffset(0);
         setFrameworkList([]);
     }, [delayedValue]);
-
-    const queryForFrameworks = useMemo(() => ({
-        fields: ['id', 'title', 'created_at'],
-        offset,
-        order: 'created_at',
-        limit: 10,
-        search: delayedValue.search,
-        relatedToMe: delayedValue.relatedToMe,
-    }), [delayedValue, offset]);
-
+    const analysisFrameworkVariables = useMemo(() => {
+        let isCurrentUserMember;
+        if (delayedValue.relatedToMe === 'true') {
+            isCurrentUserMember = true;
+        } else if (delayedValue.relatedToMe === 'false') {
+            isCurrentUserMember = false;
+        } else {
+            isCurrentUserMember = undefined;
+        }
+        return {
+            isCurrentUserMember,
```

Comment:
We should be able to directly pass relatedToMe as isCurrentUserMember. Can you check?